patternfly/patternfly-elements
# feat(tools): add `renderTitleInOverview` option to cem 11ty plugin

Closes #2437

## What I did

1. added `renderTitleInOverview` option to the custom-elements-manifest 11ty plugin.

## Testing Instructions

1. check the element docs pages on the DP, there should be no change, since the option defaults to true.
